src/core/_root.scss

```diff
@@ -1,9 +1,9 @@
 :root {
-	--accent: 98, 205, 254; /* The colour used throughout the app | R,G,B format | Default: 98, 205, 254 */
+	--accent: '98, 205, 254'; /* The colour used throughout the app | Default: 98, 205, 254 */
```

Comment:
This needs to be `rgb(98, 205, 254)`. Otherwise CSS won't recognise it as a colour unit.

Reply:
Right! Fixed